packages/bundler/localconfig/bundler.config.json

```diff
@@ -7,8 +7,11 @@
   "minBalance": "1",
   "mnemonic": "./localconfig/mnemonic.txt",
   "maxBundleGas": 5e6,
-  "minStake": "1" ,
-  "minUnstakeDelay": 0 ,
+  "eipParams":
```

Comment:
Better placed in a separate file, so that this file can be a reference by itself, regardless of bundler-specific params.